intro/calc.test.js

```diff
@@ -0,0 +1,34 @@
+const Calculator = require("./calc");
```

Comment:
Class is a blueprint to create objects. Can you create an object and then run tests on that object?